tools/create-package-components.js

```diff
@@ -79,22 +72,15 @@ async function createComponent({ componentName, packageDir }) {
   }
 
   if (!initialIsCapital(componentName)) {
-    throw new Error(
-      `Wrong format for '${componentName}': use CamelCase for ComponentName`
-    )
+    throw new Error(`Wrong format for '${componentName}': use CamelCase for ComponentName`)
   }
 
-  await fs.writeFile(
-    path.join(packageDir, 'src', `${componentName}.js`),
-    componentTemplate({ componentName })
-  )
+  await fs.writeFile(path.join(packageDir, 'src', `${componentName}.js`), componentTemplate({ componentName }))
 }
 
 function getIndexFile(componentNames) {
   return componentNames
-    .map(
-      componentName => `export ${componentName} from './src/${componentName}'`
```

Comment:
This syntax was invalid 🤔 so it should work as expected now (re-exporting a default export -> named)